src/index.js

```diff
@@ -226,6 +229,7 @@ document.querySelector("#sign").addEventListener(
           signature,
           signingScheme,
           from: await signer.getAddress(),
+          isLiquidityOrder: document.querySelector("#isLiquidityOrder").checked ? true : undefined,
```

Comment:
> Are we not worried that someone could make a users order a liquidityOrder (effectively waiving the right for surplus) without requiring signed consent?

Great catch! I hadn't thought about it to be perfectly honest.

> If we consider the scenario where a user hands their order to a third party which is responsible for forwarding it

A more nefarious thing to do would be to take, for example, a staging order, and then add it to the prod database with the same signature with `is_liquidity_order` set, which would cause the user order to never get any surplus. You could use this if you wanted to take the opposite position of the user order as a way of shifting all the surplus to you.